### PR TITLE
Define Attributes

### LINKED ISSUE #28: Definiton of an attribute is missing
Neither tool_interface.rst, nor view_component_template.rst has a definition of the attributes of a view-component

### PR DESCRIPTION
**Pull request by TheOneAndra**
Fix #28 

**Changes proposed in this pull request**:
- Open discussion on defining **Attributes**

The following steps were realized, as well (if applies):
- [x] Update the CHANGELOG.md
- [ ] Adapt certain view_component to match the defintion

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

File: docs/tool_interface.rst [hunk 1]
```diff
@@ -47,7 +47,9 @@ widget
     Smaller window within the tool's :ref:`main-window-def`.
     A *widget* can be moved around by the user within the :ref:`main-window-def` and collapsed into another *widget* (then each *widget* is accessible via tabs).
 
-
+Attribute
+---------
+    The attributes are the inherent features of a component of a view. The elements listed under **Attributes** are visible element that the user can see on screen. They can have certain *requirements* and *actions*.
```
Comment (Bachibouzouk) [inline docs/tool_interface.rst]:
```suggestion
    The attributes contain the essential information needed to characterize and render a :ref:`view-def` or a :ref:`view-component-def`. Some attributes might be rendered themselves in the :ref:`view-def` or :ref:`view-component-def` but it is not required. Some attributes can be bound to :ref:`requirement-def` of :ref:`action-def`.
```

Comment (Bachibouzouk) [reply docs/tool_interface.rst]: Also add a `.. _attribute-def` above the `Attribute` title

Comment (Bachibouzouk) [reply docs/tool_interface.rst]: I would place this definition after `view-rendering`

**Review by Bachibouzouk — APPROVED** [review]

File: docs/tool_interface.rst [hunk 2]
```diff
@@ -21,6 +21,12 @@ view-component
 --------------
     A *view-component* is a part of a *view* which can be described independently from the view and could be reused in different *views* (a menu bar for example). A *view-component* is described in terms of its attributes and its actions. For example a menu bar has a nested list of items (each item can itself be a list of item: submenus). Each item, if not itself a list, consists of a label (what the user will see) and an action (what will be done upon clicking/selecting the item.)
 
+.. _attribute-def:
+
+Attribute
+---------
+    The attributes contain the essential information needed to characterize and render a :ref:`view-def` or a :ref:`view-component-def`. Some attributes might be rendered themselves in the :ref:`view-def` or :ref:`view-component-def` but it is not required. Some attributes can be bound to :ref:`requirement-def` of :ref:`action-def`.
+
```
Comment (TheOneAndra) [inline docs/tool_interface.rst:29]: So we will define requirements and actions as well?

**Pull request merged**